canonical/ubuntu.com
# Search results page, layout and pagination

## Done

- Applied design updated to search results page
- Updated pagination functionality
- Added ordering functionality
**Note: Filtering functionality to be covered in separate PR**

## QA

- View the site locally in your web browser at: https://ubuntu-com-13440.demos.haus/security/cves
- Click on any of the "By Ubuntu release" links and check results page against design
    - Be sure to test on mobile, tablet and desktop screen sizes
- Change sorting option and see that you are returned the expected order
- Test pagination, change the result limit and see that this works as expected 
- Click on the "Export to JSON" link and see that you can access the correct api endpoint url

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-5649
